packages/polling-controller/CHANGELOG.md

```diff
@@ -6,4 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/MetaMask/core/
+## [0.0.1]
```

Comment:
0.x as a version denotes a "pre-production" release which is highly subject to change, and I feel that we would only do that if we are not confident about the readiness of the code. But, if we _do_ feel confident, what are our thoughts on using 1.0.0 instead of 0.0.1?

Reply:
So this package is pre-production in the sense that I don't expect we will be using it's functionality just yet. What do we think about versioning here @shanejonas @Gudahtt

Reply:
hm yea technically it wont be used, I do like having a pre 1.0.0 release since we might change some of the interface before a true 1.0.0, like we did already with `startPollingByNetworkClientId` method name.

Reply:
Yeah it feels like there may still be some changes to this mixin/abstract class needed before it goes to prod.

Reply:
Oh okay. What do we feel about 0.1.0 then? Where the second number sort of takes the place of major and the third number takes the place of minor.

Reply:
Yep sounds good